WTWSMS/decisions/GR_Scythian.txt

```diff
@@ -203,7 +203,7 @@ decisions = {
 	}
 }
 
-targetted_decisions = {
+decisions = {
```

Comment:
Most of the features can be found easily in the Decisions menu. Some of them cannot. I thought, for a long time, that the Convert to Local Culture and Religion buttosn were removed from the Mod, until I discovered them by accident.

For the Scythians, it's especially wrong, because consulting the Scythian Shaman is similar to Worshipping the Ancestors and issuing a Great Sacrifice, and thus, should go THERE in the Decisions menu.